data/json/itemgroups/Food/food.json

```
@@ -851,7 +851,7 @@
       { "prob": 45, "group": "cheese_bag_plastic" },
       { "item": "mozzarella_cheese", "prob": 30 },
       { "item": "ceramic_plate", "prob": 30 },
-      { "item": "knife_steak", "prob": 45 },
+      { "item": "knife_small", "prob": 45 },
```

```suggestion
      { "item": "knife_small", "variant": "knife_steak", "prob": 45 },
```
You'll want to make sure most of these have the same chances of spawning a steak or paring knife appropriately, a paring knife isn't usually something you'd see in a dining set for example.